package com.sec.car_selling.dto.request;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class CalculateInstallmentRequest {

    int loanYear;

    BigDecimal downPayment;

    BigDecimal price;

    BigDecimal bankInterest;
}
